virt/virtual_machines/virtual_disks/virt-configuring-shared-volumes-for-vms.adoc

```diff
@@ -11,7 +11,7 @@ You can configure shared disks to allow multiple virtual machines (VMs) to share
 You configure disk sharing by exposing the storage as either of these types:
 
 * An ordinary VM disk
-* A logical unit number (LUN) disk with an iSCSi connection and raw device mapping, as required for Windows Failover Clustering for shared volumes
+* A logical unit number (LUN) disk with an SCSi connection and raw device mapping, as required for Windows Failover Clustering for shared volumes
```

Comment:
Is it SCSI, like in the module, or SCSi, like here?

Reply:
That's definitely a typo that I missed. Thank you for catching it out!